modules/core/server/views/layout.server.view.html

```diff
@@ -65,6 +65,11 @@
 
   <!--Application JavaScript Files-->
     {{#each jsFiles}}<script type="text/javascript" src="{{this}}"></script>{{/each}}
+  
+  <!--owasp config sync-->
+  <script type="text/javascript">
+    owaspPasswordStrengthTest.configs = {{{ owaspConfig }}};
```

Comment:
Shouldn't this be `owaspPasswordStrengthTest.config`, according to https://github.com/nowsecure/owasp-password-strength-test#configuring?